package.json

```diff
@@ -10,7 +10,8 @@
     "package-all": "npm run build && build -mwl",
     "package-linux": "npm run build && build --linux",
     "package-win": "npm run build && build --win --x64",
-    "standard": "standard --fix 'app/**/*.js' && yarn run lint",
+    "precommit": "lint-staged",
```

Comment:
@sagivf to run the prettier or the standard first?

Reply:
@rodrigooler sounds right to me